src/sankey.js

```diff
@@ -51,12 +53,20 @@ function computeLinkBreadths({nodes}) {
   }
 }
 
+function orientNodes({nodes}, orientation) {
+  for (const node of nodes) {
+    [node.x0, node.y0] = orientation(node.x0, node.y0);
+    [node.x1, node.y1] = orientation(node.x1, node.y1);
```

Comment:
One note as I re-read this: these lines use ES6 destructuring ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), [adoption @ 89.91% currently](https://caniuse.com/#search=destructuring)).

Perhaps it would be safer / more compatible if I switched this to simply use straightforward array accesses.